### PR TITLE
updates changelog

### PR DESCRIPTION
**Pull request by BoDonkey**
## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
Updates the CHANGELOG with a description of the widget placeholders feature since the feature's documentation is spread across multiple pages.

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor
- [X] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [ ] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [X] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: CHANGELOG.md
Comment (ETLaurent) [inline CHANGELOG.md:7]:
```suggestion
* Adds the widget placeholders feature to the core and custom widgets. This includes toggling placeholder content when adding widgets to an area, replacing default content for core widgets, and adding placeholder content for multiple schema fields.
```

Something like that, to emphasize the fact they placeholders are there when adding the widgets,  what do you think?

**Review by ETLaurent — APPROVED** [review]

File: CHANGELOG.md
Comment (BoDonkey) [reply CHANGELOG.md:7]: They haven't been accepted, yet, so I can't add specific links. How about I add something like:
"See the documentation in the Guide section for 'Areas and Widgets' for further details."?

Comment (ETLaurent) [reply CHANGELOG.md:7]: Hmm that I wanted to do in the first place but @boutell  told me that we should have the link, and I agree, for the sake of a better user experience (having the link and not having to look for the documentation is quite cool, because we're all lazy, especially developers 😄 )

We could guess what the link will  be from the title of the documentation page, eg:

```
https://v3.docs.apostrophecms.org/guide/caching.html
```

but that's risky if the page title changes before the merge.
I'd  say we wait for the doc to be merged first :)

Comment (ETLaurent) [reply CHANGELOG.md:7]: I checked with https://github.com/apostrophecms/a3-docs/blob/PRO-3236-widget-placeholders/docs/guide/* repo and it seems to be fine, if the titles don't change I guess it's ✅

**Comment by boutell** [comment]
Just spotted this. I'm going to close because I did the same in the release-3.31.0 PR, which is pending approval by Miro and then we'll have a release. I went with simpler language saying that the placeholder: true and initialModal: false features offer ways to improve the user experience of adding widgets to areas, and I linked to the correct section of the docs, which have been merged and published.

**Pull request closed without merging**
